createdisk.sh

```diff
@@ -149,6 +149,20 @@ function update_json_description {
         >$destDir/crc-bundle-info.json
 }
 
+function eventually_add_pull_secret {
+    local destDir=$1
+
+    if [[ -f "$BUNDLED_PULL_SECRET_PATH" ]]
+    then
+      cat "$BUNDLED_PULL_SECRET_PATH" > "$destDir/default-pull-secret"
+      mv $destDir/crc-bundle-info.json $destDir/crc-bundle-info.json.tmp
+      cat $destDir/crc-bundle-info.json.tmp \
+          | ${JQ} '.clusterInfo.openshiftPullSecret = "default-pull-secret"' \
+          >$destDir/crc-bundle-info.json
+      rm $destDir/crc-bundle-info.json.tmp
```

Comment:
```
cat $destDir/crc-bundle-info.json ... >$destDir/crc-bundle-info.json.tmp
mv $destDir/crc-bundle-info.json.tmp $destDir/crc-bundle-info.json
```
would save the need for the `rm`, but that's not really important

Reply:
done, thanks!